azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Manage syslog forwarding for Azure Stack HCI (preview)
+description: Learn about security features for Azure Stack HCI security information and event management (SIEM) (preview).
+author: alkohli
+ms.author: alkohli
+ms.topic: how-to
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 12/01/2023
+---
+
+# Manage syslog forwarding for Azure Stack HCI (preview)
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article describes how to manage components in the security layer of the Azure Stack HCI, version 23H2 solution for security information and event management (SIEM).
+
```

Comment:
Rewrite to indicate that are you describing how to configure syslog forwarding .....

Reply:
Also 1 sentence about syslog.

Reply:
Done > updated to indicate "configuring" and added sentence about syslog.